src/utils/components/ConfirmActionMessage/ConfirmActionMessage.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { Trans } from 'react-i18next';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
+
+const ConfirmActionMessage: React.FC<{ obj: K8sResourceCommon; action?: string }> = ({
+  obj,
```

Comment:
I think it might be a little cleaner to accept `name` and `namespace` props instead of building custom objects when you use the components. What do you think?

Reply:
Well, this depends on where the component is used. I'd leave it as it is, as we need a custom object only in a few cases.

Reply:
The component doesn't need a `K8sResourceCommon` object; it needs a `name` and a `namespace`. Passing only what is needed would make both the component and the places it's used cleaner. Also, the `namespace` prop would be optional, which would help in making the component's interface even clearer.

Reply:
From my perspective, it's better to send `K8sResourceCommon` object as a prop to the component - as it was originally done. I believe this was done with a good purpose. Also the component takes care by itself about what is needed from the object. If we did it the different way, we would have to extract the name and the namespace - to do the same stuff in all the places where the component is used - I consider this unnecessary.

Reply:
Having `K8sResourceCommon` as a prop falsely implies that the component only works for `K8sResourceCommon` objects, which isn't true. Maybe it would have been better to just create a `ConfirmDetachMessage` component and leave this one as it was originally.

Reply:
Thank you, Phillip, this explanation did persuade me to think that we don't want `K8sResourceCommon` as a prop type :)
 
Yes, I was also thinking about creating a separate component for "detach" message, not to mix different types of message components, but I'm not sure if it worth, as the messages are almost the same. I need to re-think this. If you have any other ideas about this or opinions what's a better solution, let me know. Thank you very much! :)

Reply:
I think the right solution hinges on whether or not there is a good chance that other data from the object will be needed in the future. If yes, then it's cleaner IMO to create a separate component for detach, since there is no `K8sResourceCommon` object used in those cases. If there's not, then I think it's cleaner to use a single component that takes only the props needed to function.

That said, if you decide you still want to keep it the way it is, I'll lgtm it. Just let me know.